news/3674.feature

```diff
@@ -0,0 +1 @@
+Activate duplication feature if not already active
```

Comment:
It seems a `breaking` change to me, especially if some customer does not want that.

If this has to be included for everyone, I would just remove the feature flag.

Reply:
changed to breaking.